pages/register/register.view.scss

```diff
@@ -1,5 +1,6 @@
 .register-page {
 	@extend %align-items-center;
+	background-color: $mainColor;
```

Comment:
This background-color overwrite register_bg, line below? Do we separate background login and register for vimojo (images resources) and elconfidencial (colors).

Reply:
No, it does not override the `background-image`, but gets combined with it. If there's an image, will use the image. If not (or while the image is loading), will use the color.